subiquity/models/locale.py

```diff
@@ -36,7 +36,6 @@ class LocaleModel(object):
         ('fr_FR', 'French'),
         ('de_DE', 'German'),
         ('el_GR', 'Greek, Modern (1453-)'),
-#        ('he_IL', 'Hebrew'), # disabled as it does not render correctly on a vt with default font
```

Comment:
I want to keep this one for when we start using bterm or something.

Reply:
That's fine, we can add a 'noqa:  after the second hash before your disabled comment.

Reply:
Do you want to do that and then merge this branch then?